search/urls.py

```diff
@@ -1,16 +1,15 @@
 """ expose courseware search http interface """
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
 from . import views
 
 COURSE_ID_PATTERN = getattr(settings, "COURSE_ID_PATTERN", r'(?P<course_id>[^/+]+(/|\+)[^/+]+(/|\+)[^/]+)')
 
 # urlpatterns is the standard name to use here
 # pylint: disable=invalid-name
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^$', views.do_search, name='do_search'),
     url(r'^{}$'.format(COURSE_ID_PATTERN), views.do_search, name='do_search'),
     url(r'^course_discovery/$', views.course_discovery, name='course_discovery'),
```

Comment:
Unrelated, but does anyone know what this course_discovery is used for?  Is it used by Open edX, or not at all? It was [added here](https://github.com/edx/edx-search/commit/35020548f27e87b663ed07bcd8de5f1d9da2abe1).  The only usages of edx-search I was aware of was course search (but I _think_ that is different), and team search.